### DISCUSSION TITLE
Clang frontend command failed due to signal compilation error

### DISCUSSION THREAD

**Discussion by souravmat-git** [idaholab/moose#17787] [Q&A Getting Started] [answered]
Hello,

After installing MOOSE using the Conda environment, I am getting the following error while compiling in MacOS BigSur 11.3.1

<img width="1440" alt="Error" src="https://user-images.githubusercontent.com/29055022/117657359-17b80700-b1b7-11eb-8f74-3cf27f0b3e83.png">

Previously, I was not getting this error. It occurred once I upgraded my macOS.
Any suggestions are welcome.

Thanks,
Sourav

**Comment by milljm** [comment]
You may have better luck by following this thread: #17722 

The gist of the _possible_ solution, is to downgrade (or possibly reinstall... I had to reinstall) Xcode 12.4, run Xcode (to finalize the installation), and then attempt at rebuilding your Application /or MOOSE.

**Reply by souravmat-git** [reply]
Thanks Jason, it works.

**Comment by cticenhour** [comment, marked as answer]
For those who experience a similar issue in the future, I am re-answering this question now that MOOSE supports Xcode 12.5, 13, and 13.1. Please update your conda packages to the newest versions by performing 
```
conda update --all
```
within your MOOSE conda environment. See #19289 for more information.
